packages/suite/src/views/dashboard/AssetsView/AssetTable/AssetStakingRow.tsx

```diff
@@ -0,0 +1,64 @@
+import { isTestnet } from '@suite-common/wallet-utils';
+import { Column, Icon, Table } from '@trezor/components';
+import { spacings, borders } from '@trezor/theme';
+import { CoinBalance, HiddenPlaceholder, Translation } from 'src/components/suite';
+import styled, { css } from 'styled-components';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { FiatValue } from 'src/components/suite';
+import { Text } from '@trezor/components';
+
+const Section = styled.div<{ $renderBothRows: boolean }>`
```

Comment:
Can you please abstract this into a separate file and use it in all three types of rows?

Reply:
https://github.com/trezor/trezor-suite/pull/14996/commits/2ebf646952b19aed134aa8613c51dd5d924c5fb1 
Thanks for a good idea